borg/remote.py

```diff
@@ -166,7 +166,7 @@ def __init__(self, location, create=False, exclusive=False, lock_wait=None, lock
         self.preload_ids = []
         self.msgid = 0
         self.to_send = b''
-        self.cache = {}
+        self.get_cache = {}
```

Comment:
I prefer explicit names. But thinking about this, it isn’t really a cache. Maybe it should be preload_msgids. After all it’s a mapping from chunkid to a list of message ids for the get requests. (list because if a chunk is preloaded multiple times it just requests it again each time, because RemoteRepository.responses has no way of "ref counting")